modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/welcome-banner/index.css

```diff
@@ -9,6 +9,42 @@
 	right: 0;
 }
 
+.svg-style {
+	margin-bottom: 4px;
+}
+
+.text-description {
```

Comment:
Check the standardization of CSS classes

note: ".customer-portal-new"

Reply:
Corrected